### PR TITLE
update interpolation argument documentation

### PR DESCRIPTION
**Pull request by tjhei**
also see  #547

### CODE REVIEW

File: source/world_builder/features/interface.cc [hunk 1]
```diff
@@ -89,8 +88,8 @@ namespace WorldBuilder
 
                 prm.declare_entry("interpolation",Types::String("global"),
                                   "What type of interpolation should be used to enforce the minimum points per "
-                                  "distance parameter. Options are global, none, linear, monotone spline and "
-                                  "continuous monotone spline interpolation. If this "
+                                  "distance parameter. Options are 'global' and "
+                                  "'continuous monotone spline' interpolation. If this "
```
Comment (MFraters) [inline source/world_builder/features/interface.cc:92]: Could you add a note here that this parameter is deprecated?

**Review by MFraters — APPROVED** [review]
one small comment, otherwise good to go.

File: source/world_builder/features/interface.cc (hunk 1 above)
Comment (tjhei) [reply source/world_builder/features/interface.cc:92]: is this because the global one is also only "continuous monotone spline"?

Comment (MFraters) [reply source/world_builder/features/interface.cc:92]: yes, and I probably want to get rid of this option before the 1.0 release.

**Comment by tjhei** [comment]
updated.

**Pull request merged**